CMakeLists.txt

```diff
@@ -405,6 +405,15 @@ add_custom_target(check-integration
     USES_TERMINAL
     VERBATIM)
 add_dependencies(check-integration test-gravity)
+add_custom_target(check-themes
+    tests/themes/run.sh
+    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+    COMMENT "Testing themes"
+    USES_TERMINAL
+    DEPENDS generated_icons generate_awesomerc ${PROJECT_AWE_NAME}
+    VERBATIM)
+add_dependencies(check-integration check-themes)
```

Comment:
I don't really like "only on Travis" tests. If I tell CMake "run tests", I want it to run tests and not be surprised 10 minutes later when the build on Travis fails.

If `check-themes` fails, then tests failed. I agree that it might be nice to get all test failures, but then people will have to use `make -k`.